packages/schematics/src/collection/ngrx/schema.json

```diff
@@ -6,34 +6,35 @@
   "properties": {
     "name": {
       "type": "string",
-      "description": "Name of the ngrx feature (e.g., Products, User, etc.).",
+      "description": "Name of the NgRx feature (e.g., Products, Users, etc.). Recommended to use plural form for name.",
       "$default": {
         "$source": "argv",
         "index": 0
       }
     },
+    "directory": {
+      "type": "string",
+      "default": "+state",
+      "description":
+        "Override the name of the folder used to contain/group the NgRx files: contains actions, effects, reducers. selectors. (e.g., +state)"
+    },
     "module": {
       "type": "string",
+      "default": "",
       "description":
-        "Path to ngModule; host directory will contain the new '+state' directory (e.g., src/libs/mylib/mylib.module.ts)."
+        "Path to ngModule; host directory will contain the new '+state' directory (e.g., src/libs/comments/state/lib/comments-state.module.ts)."
```

Comment:
this path won't exist an nx workspace. Use a real path (e.g., libs/comments/src/lib/state/comments-state.module.ts).

Reply:
Fixed.